apps/events/models/event.py

```diff
@@ -0,0 +1,47 @@
+from django.db import models
+
+from .constants import (
+    COMMUNITIES,
+    EVENT_CATEGORIES,
+    EVENT_TYPES,
+    Community,
+    EventCategory,
+    EventType,
+)
+from .speaker import Speaker
+
+
+class Event(models.Model):
+    category = models.CharField(
+        max_length=50, choices=EVENT_CATEGORIES, default=EventCategory.WORKSHOPS
+    )
+    for_community = models.CharField(
+        max_length=50, choices=COMMUNITIES, default=Community.DJANGO_CAMEROON
+    )
+    title = models.CharField(max_length=100)
+    description = models.TextField()
+    location = models.CharField(max_length=100)
+    hour = models.TimeField()
+    date = models.DateField()
+    type = models.CharField(
+        max_length=50, choices=EVENT_TYPES, default=EventType.IN_PERSON
+    )
+    speaker = models.ForeignKey(Speaker, on_delete=models.CASCADE)
+    tags = models.ManyToManyField("EventTag", related_name="events", default=None)
+    # TODO: Add field for displying who created the event
+    # created_by = models.ForeignKey(User, on_delete=models.CASCADE)
+    published = models.BooleanField(default=False)
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
For such fields I suggest we have an inherited model to handle all this. Something like:
```python
class BaseModel(models.Model):
    """
    Abstract model that provides self-updating ``created`` and ``modified``
    fields.
    """

    id = models.UUIDField(
        _("id"),
        primary_key=True,
        default=generate_uuid,
        editable=False,
        help_text=_("Unique identifier for this object."),
    )
    created = models.DateTimeField(
        _("created"),
        auto_now_add=True,
        help_text=_("Date and time when this object was created."),
    )
```